### PR TITLE
Kubernetes migration complete

### PR DESCRIPTION
**Pull request by Akvanvig**

### CODE REVIEW

File: Deployment_files/kube-deployment_github [hunk 1]
```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Pod
```
Comment (audunsolemdal) [inline Deployment_files/kube-deployment_github:2]: delete this

File: Deployment_files/test-3_kube-pod [hunk 2]
```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Pod
```
Comment (audunsolemdal) [inline Deployment_files/test-3_kube-pod:2]: no pods just deployments , replicasets or statefulsets.......

Comment (MrOctopus) [reply Deployment_files/test-3_kube-pod:2]: Hi Audun!

Have you tried making a pull request?

File: Deployment_files/3_database-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: roboto-postgres
+  namespace: discord-bot-py
+  labels:
+    app: roboto-postgres
+spec:
+  replicas: 1
```
Comment (audunsolemdal) [inline Deployment_files/3_database-deployment.yaml:9]: where are the requests and cpu limits at

File: Deployment_files/4_kube-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    matchLabels:
+      app: roboto
+  template:
+    metadata:
+      labels:
+        app: roboto
+    spec:
+      containers:
+      - name: roboto
+        image: ghcr.io/akvanvig/discord-bot-py_no-audio:latest
```
Comment (audunsolemdal) [inline Deployment_files/4_kube-deployment.yaml:20]: :latest bruh.gif

File: Deployment_files/test-3_kube-pod [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      configMap:
+        name: discord-py-config
+    - name: secrets
+      secret:
+        secretName: discord-py-secrets
+    - name: audiofiles
+      persistentVolumeClaim:
+        claimName: nfs-robotoaudiofiles
+
+  restartPolicy: Never
```
Comment (audunsolemdal) [inline Deployment_files/test-3_kube-pod:37]: sure about "never"?

File: Deployment_files/test-3_kube-pod (hunk 2 above)
Comment (audunsolemdal) [reply Deployment_files/test-3_kube-pod:2]: Please revert the PR, it causes instability to the main branch!!!

File: Deployment_files/test-3_kube-pod (hunk 5 above)
Comment (Akvanvig) [reply Deployment_files/test-3_kube-pod:37]: Only used for testing, not in prod ¯\_(ツ)_/¯

File: Deployment_files/4_kube-deployment.yaml (hunk 4 above)
Comment (Akvanvig) [reply Deployment_files/4_kube-deployment.yaml:20]: Prod is test, why not

File: Deployment_files/kube-deployment_github (hunk 1 above)
Comment (Akvanvig) [reply Deployment_files/kube-deployment_github:2]: Only for testing, not deployed otherwise

File: Deployment_files/test-3_kube-pod (hunk 2 above)
Comment (MrOctopus) [reply Deployment_files/test-3_kube-pod:2]: I will forward the request to our QA department

File: Deployment_files/3_database-deployment.yaml (hunk 3 above)
Comment (Akvanvig) [reply Deployment_files/3_database-deployment.yaml:9]: I'm taking this up at the next CAB advisory board meeting.
Thank you for your input!

**Pull request merged**